docs/proposals/201812_thanos-remote-receive.md

```diff
@@ -106,23 +106,40 @@ Hard tenants in the Thanos receiver are configured in a configuration file. Chan
 
 A sample of the configuration of tenants and their respective infrastructure:
 
-```json
-[
-    {
-        "hashring": "tenant-a",
-        "endpoints": ["tenant-a-1.metrics.local:19291/api/v1/receive", "tenant-a-2.metrics.local:19291/api/v1/receive"],
-        "tenants": ["tenant-a"]
-    },
-    {
-        "hashring": "tenants-b-c",
-        "endpoints": ["tenant-b-c-1.metrics.local:19291/api/v1/receive", "tenant-b-c-2.metrics.local:19291/api/v1/receive"],
-        "tenants": ["tenant-b", "tenant-c"]
-    },
-    {
-        "hashring": "soft-tenants",
-        "endpoints": ["http://soft-tenants-1.metrics.local:19291/api/v1/receive"]
-    }
-]
+```yaml
+route:
+- hashring_group: soft-tenants
+  partition_keys:
+  - tenant_id
+  routes:
```

Comment:
having `route` at the top level and `routes` here is a bit confusing. how about:
```suggestion
  target:
```

Reply:
the point here is that routes are nest-able, much like the alertmanager routing tree

Reply:
got it, some commenting would be nice to avoid the confusion I guess.

Reply:
yes makes sense